test/quantization/core/test_quantized_tensor.py

```diff
@@ -861,19 +944,6 @@ def test_choose_qparams(self, X, reduce_range):
         np.testing.assert_array_almost_equal(X_scale, qparams[0], decimal=3)
         self.assertEqual(X_zp, qparams[1])
 
-    @unittest.skipIf(not torch.cuda.is_available() or TEST_WITH_ROCM, 'CUDA is not available')
```

Comment:
This test seems to be similar to the numerics check that you have earlier, and only checks for approximate equality.

Reply:
it tests that the int representations are equal but the decimal representations are not required to be. I can certainly make the change though.